src/layouts/Homepage.js

```diff
@@ -1,28 +1,48 @@
 import React from 'react'
 import PropTypes from 'proptypes'
+import styled from 'styled-components'
 
 import Container from './Container'
+import { Link } from 'phenomic'
 
-import HeroSection from '../components/HeroSection'
-import ProductSection from '../components/ProductSection'
-import DataSection from '../components/DataSection'
-import PrivacySection from '../components/PrivacySection'
-import ContactSection from '../components/ContactSection'
+const Boarding = styled.div`
+	background: linear-gradient(to bottom, #2dd7d3, #8c7ddd);
+`
 
-const HomePage = (props) => (
-    <Container { ...props }>
-        <div>
-            <HeroSection { ...props.head.sections.hero }/>
-            <ProductSection { ...props.head.sections.product }/>
-            <DataSection { ...props.head.sections.data }/>
-            <PrivacySection { ...props.head.sections.privacy }/>
-            <ContactSection { ...props.head.sections.contact }/>
-        </div>
-    </Container>
-)
+const Logo = styled.span`
+		    padding: 1rem 2rem;
+				display: inline-block;
+				&:hover,
+				&:active,
+				&:focus {
+						transform: scale(1.05);
+				}
+				img {
+						width: auto;
+						height: 2rem;
+				}
+`
+
+const HomePage = props => (
+	<Container {...props}>
+		<Boarding>
+			<Logo>
+					<Link to="/">
+							<img src="/assets/logo.svg" alt="Cinuru" />
+					</Link>
+			</Logo>
+			<iframe
+				src="http://85.214.78.13:3000/"
```

Comment:
don't put anything but the form in the iframe. (like the `Some content goes here.` text) and try to make the form in the iframe just grow to 100% width with all its input elements. Then you can controll the width of the frame from here. Also why does the form as for first and last name, do we actually need that? Otherwise I'd prefer a simple name input

Reply:
And it would be preferable to make the iframe also use Roboto